llama-index-integrations/llms/llama-index-llms-mistralai/llama_index/llms/mistralai/base.py

```diff
@@ -444,3 +445,19 @@ def get_tool_calls_from_response(
             )
 
         return tool_selections
+
+    def fill_in_middle(
+        self, prompt: str, suffix: str, stop: List[str] = ["\n\n"]
```

Comment:
Best practice not to use lists or dicts as default arguments (it can have unexpected consequences due to how pass-by-reference works)